core/server/worker/src/main/java/alluxio/worker/modules/DoraWorkerModule.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.modules;
+
+import alluxio.ClientContext;
+import alluxio.client.file.cache.CacheManager;
+import alluxio.client.file.cache.CacheManagerOptions;
+import alluxio.client.file.cache.store.PageStoreDir;
+import alluxio.conf.AlluxioConfiguration;
+import alluxio.conf.Configuration;
+import alluxio.conf.PropertyKey;
+import alluxio.master.MasterClientContext;
+import alluxio.network.TieredIdentityFactory;
+import alluxio.underfs.UfsManager;
+import alluxio.underfs.WorkerUfsManager;
+import alluxio.wire.TieredIdentity;
+import alluxio.worker.WorkerFactory;
+import alluxio.worker.block.BlockStore;
+import alluxio.worker.dora.DoraWorker;
+import alluxio.worker.dora.DoraWorkerFactory;
+import alluxio.worker.dora.PagedDoraWorker;
+import alluxio.worker.file.FileSystemMasterClient;
+import alluxio.worker.page.PagedBlockMetaStore;
+import alluxio.worker.page.PagedBlockStore;
+import alluxio.worker.page.PagedBlockStoreDir;
+
+import com.google.inject.AbstractModule;
+import com.google.inject.Scopes;
+import com.google.inject.TypeLiteral;
+import com.google.inject.name.Names;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Guice module for block worker.
+ */
+public class DoraWorkerModule extends AbstractModule {
+  @Override
+  protected void configure() {
+    bind(TieredIdentity.class).toProvider(() ->
+        TieredIdentityFactory.localIdentity(Configuration.global()));
+    bind(new TypeLiteral<AtomicReference<Long>>() {
+    }).annotatedWith(Names.named("workerId"))
+        .toInstance(new AtomicReference<>(-1L));
+    bind(FileSystemMasterClient.class).toProvider(() -> new FileSystemMasterClient(
+        MasterClientContext.newBuilder(ClientContext.create(Configuration.global())).build()));
+    bind(UfsManager.class).to(WorkerUfsManager.class).in(Scopes.SINGLETON);
```

Comment:
For dora we need `DoraUfsManager` which does not talk to master to fetch mount info.